lucuma-schemas/src/main/scala/lucuma/schemas/odb/ExecutionVisitsSubquery.scala

```diff
@@ -9,16 +9,30 @@ import lucuma.schemas.decoders.given
 import lucuma.schemas.model.ExecutionVisits
 
 object ExecutionVisitsSubquery
-    extends GraphQLSubquery.Typed[ObservationDB, ExecutionVisits]("Execution"):
+    extends GraphQLSubquery.Typed[ObservationDB, Option[ExecutionVisits]]("Execution"):
   override val subquery: String = s"""
```

Comment:
@rpiaggio Are you sure this was working for observations with no sequence? Even prior to these API changes the `config` would be `null` and the decoder should still have failed, AFAICT.

Reply:
No, I'm not sure it was working in those cases.